xdsl/backend/mps/print_mps.py

```diff
@@ -0,0 +1,12 @@
+from typing import IO
+
+from xdsl.dialects.builtin import ModuleOp
+
+
+def print_to_mps(prog: ModuleOp, output: IO[str]):
+    # TODO: implement lowering to proprietary AIR format
+    # see: https://github.com/sueszli/llvm-to-air
+    # demo:
+    # $ uv run xdsl-opt -t mps tests/filecheck/backend/mps/module.mlir
+    # $ uv run lit tests/filecheck/backend/mps/module.mlir
```

Comment:
most of this should move into the docstring of the module, except that I would recommend avoiding the link to the other repo, since my understanding is that we're going to move its contents here (or to xdsl-mps in the future, but that'll just be this code we're adding now anyway)